Microsoft.Toolkit.Uwp.UI.Controls/Menu/MenuItem.cs

```diff
@@ -111,6 +111,11 @@ protected override void OnApplyTemplate()
                 MenuFlyout.Closed -= MenuFlyout_Closed;
             }
 
+            if (XamlRoot != null)
```

Comment:
>XamlRoot [](start = 16, length = 8)

At various points in this PR you're checking the XamlRoot property for null - but that won't work downlevel (the QI will fail before it ever reads the property).  I think this will cause crashes.
You'll need to use ApiInformation to only check XamlRoot when "uplevel enough".

Reply:
Addressed by https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3206/commits/92f8328fd47b9731f239a161e037aa77956e6f3d